README.md

```diff
@@ -1,76 +1,94 @@
-meta-parsec layer
-==============
+# meta-parsec layer
 
 
-This layer contains recipes for the Parsec service with either a software TPM or a hardware TPM or a PKCS11 provider service.
+This layer contains recipes for the Parsec service with either a hardware TPM or a software TPM or a PKCS11 provider service.
 
+## Hardware TPM Provider
 
-Software TPM Password
-============
+### Hardware TPM Password
 
-By default both the parsec and software TPM are configured to use the password **tpm_pass**  
-This **MUST** be changed as part of the factory setup process.  
+By default parsec and hardware TPM are configured to use an empty password  
 
 To change this perform the following steps:
 1. Stop the parsec service  
 ```sudo systemctl stop parsec```
```

Comment:
This should be `something` if it's in one line, otherwise it needs to be 

formatter (the three ' things ), line change
text
formatter (the three things).

Reply:
I'm guessing GitHub MD preview will not handle those correctly.

Reply:
Updated throughout the document.